README.md

```diff
@@ -588,9 +588,60 @@ Run [`npx nvmrc`](https://npmjs.com/nvmrc) to validate an `.nvmrc` file. If that
 
 ### Deeper Shell Integration
 
-You can use [`avn`](https://github.com/wbyoung/avn) to deeply integrate into your shell and automatically invoke `nvm` when changing directories. `avn` is **not** supported by the `nvm` maintainers. Please [report issues to the `avn` team](https://github.com/wbyoung/avn/issues/new).
+You can use [`nvshim`](https://github.com/iamogbz/nvshim) to shim the `node`, `npm`, and `npx` bins to automatically use the `nvm` config in the current directory. `nvshim` is **not** supported by the `nvm` maintainers. Please [report issues to the `nvshim` team](https://github.com/iamogbz/nvshim/issues/new).
```

Comment:
i'm not sure this is a tool i want to recommend - reading the readme, it seems like it breaks `nvm use` instead of just auto-using the local version?

Reply:
Sorry for the confusion, I actually didn't add the `nvshim` recommendation in this line, i just removed the word "also" at the beginning. This tool was recommended back in #2193.

Reply:
ah, lol, well if it's already in there then fair enough